### DISCUSSION TITLE
Enable max token length greater than 750?

### DISCUSSION THREAD

**Discussion by Jenetrix** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#169] [Q&A]
Is this a software or hardware limitation? Some of the negative prompts I am using are regularly in the ~1400 length range.

**Comment by J-Cott** [comment]
You should test if they are actually making a difference, I bet they are not at that length, a small number of Negative prompts does improve some SD 1.5 models, SDXL models don't need any.

**Reply by Jenetrix** [reply]
I’ve rewritten one specific base prompt from scratch 3 times with several different models and different versions of the same model in mind. (All 1.5 based models) The negative prompt being as large as it is was the only way I could get the output images (as well as the complex patterns in those images) to be highly consistent across the board. This is before I even add set-specific embeddings on top of that. I’ve experimented with removing parts of my negative prompt to bring it under 750 but the output images aren't as consistent without them.
